src/app/shared/services/profile/profile.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpHeaders, HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {
+  UserProfile,
+  UserProfileResponse,
+  UserProfileUpdatedResponse
+} from 'src/app/models/Profile';
+import { LocalStorageService } from '../local-storage.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+  profileUrl = 'http://127.0.0.1:8000/api/v1/auth/profile/';
```

Comment:
I believe this should not be here

Reply:
Hi, do you have a suggestion of how this can be fixed? When I move the URL to the environment file, I keep getting an error ... not sure how to go about this

Reply:
hey did you fix this @MandelaK

Reply:
I think so ...  like I said earlier, I don't really understand the change you wanted me to make, maybe you can comment on that?

Reply:
the profile url i meant should be in the config, but i just saw you did so, good work on this.